articles/synapse-analytics/sql-data-warehouse/sql-data-warehouse-tables-partition.md

```diff
@@ -234,16 +230,18 @@ AS
 SELECT  *
 FROM    [dbo].[FactInternetSales_20000101]
 WHERE   [OrderDateKey] >= 20000101
-AND     [OrderDateKey] <  20010101
-;
+AND     [OrderDateKey] <  20010101;
 
 ALTER TABLE dbo.FactInternetSales_20000101_20010101 SWITCH PARTITION 2 TO dbo.FactInternetSales PARTITION 2;
 ```
 
-Once you have completed the movement of the data, it is a good idea to refresh the statistics on the target table. Updating statistics ensures the statistics accurately reflect the new distribution of the data in their respective partitions.
+Once you have completed the movement of the data, it is a good idea to refresh the statistics on the target table and to drop unnecessary tables. Updating statistics ensures the statistics accurately reflect the new distribution of the data in their respective partitions.
 
 ```sql
 UPDATE STATISTICS [dbo].[FactInternetSales];
+DROP TABLE [dbo].[FactInternetSales_20000101]
+DROP TABLE [dbo].[FactInternetSales_20000101_20010101]
+
 ```
 
```

Comment:
Finally, in the case of a one-time partition switch to move data, you could drop the tables created for the partition switch, `FactInternetSales_20000101_20010101` and `FactInternetSales_20000101`.  Alternatively, you may want to keep empty tables for regular, automated partition switches.